src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/BrushesTests/DynamicBrushes_On_Shapes.xaml

```diff
@@ -14,7 +14,7 @@
 				<Ellipse x:Name="shape1" Width="50" Height="60" />
 				<Rectangle x:Name="shape2" Width="50" Height="60">
 					<Rectangle.Fill>
-						<ImageBrush ImageSource="ms-appx:///Assets/uno-overalls.jpg" />
+						<ImageBrush/>
```

Comment:
I think the original intent was to have default value, and that the dynamic change actually changes something.